rfcs/2023-09-12-second-relays.md

```diff
@@ -167,27 +167,33 @@ Below diagram shows the encrypttion layers for `forward` and `response` commands
 -----------------             -----------------  -- TLS --  -----------------             -----------------
 ```
 
-When proxy connects to SMP relay it would indicate in the handshake that it will use proxy protocol and the SMP relay would expect the same `forward` commands and reply with `response`s.
+When proxy connects to SMP relay it would indicate in the handshake that it will act as a proxy and the SMP relay would expect the same `forward` commands and reply with `response`s.
```

Comment:
Most likely s2r layer will be removed if entity ID and command encryption is added.

Correlation ID should be random, possible from a separate DRG per session, to minimise contention.